specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/examples/amlFilesystems_ListByResourceGroup.json

```diff
@@ -0,0 +1,132 @@
+                "timeOfDay" : "22:00"
+              },
+              "hsmSettings": {
+                "container": "https://somecontainer.blob.core.windows.net/containername",
+                "importPrefix": "fs1",
+                "exportPrefix": "fs1"
+              },
+              "namespaceHydrationStatus": {
+                "state": "NotStarted",
+                "progress": {
+                  "blobCount": 1,
+                  "blobName": "fsblob",
+                  "size": 4.4,
+                  "rate": 44.4,
+                  "errorCount": 0
+                }
+              },
+              "archiveSettings" : {
+                "filesystemPath" : "/mnt/lfs1",
+                "status" : {
+                  "state" : "InProgress",
+                  "lastCompletionTime" : "2019-04-21T18:25:43.511Z",
+                  "percentComplete" : 50
+                }
+              }
+            },
+            "sku": {
+              "name": "Standard_2G"
+            }            
+          },
+          {
+            "tags": {
+              "Dept": "ContosoAds"
+            },
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.StorageCache/amlFilesystems/fs2",
+            "location": "westus",
+            "name": "fs2",
+            "type": "Microsoft.StorageCache/amlFilesystem",
+            "properties": {
+              "storageCapacityTiB": 3072,
+              "zones": ["1", "2"],
+              "health": {
+                "state": "Healthy",
+                "statusDescription": "amlFilesystem is ok."
+              },
+              "provisioningState": "Succeeded",
+              "subnet": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.Network/virtualNetworks/amlvnet/subnets/sub2",
+              "MGSAddress" : "10.0.0.4",
+              "mountCommand" : "mount -t lustre 10.0.0.4@tcp:/lustrefs /lustre/lustrefs",
+              "lustreVersion" : "2.12.5",
+              "encryptionSettings": {
+                "keyEncryptionKey": {
+                  "keyUrl": "https://examplekv.vault.azure.net/keys/kvk/3540a47df75541378d3518c6a4bdf5af",
+                  "sourceVault": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/some-rg/providers/Microsoft.KeyVault/vaults/examplekv"
+                }
+              },
+              "maintenanceWindow" : {
+                "dayOfWeek": "Friday",
+                "timeOfDay" : "22:00"
+              },
+              "hsmSettings": {
+                "container": "https://somecontainer.blob.core.windows.net/containername",
+                "importPrefix": "fs1",
+                "exportPrefix": "fs1"
+              },
+              "namespaceHydrationStatus": {
+                "state": "NotStarted",
+                "progress": {
+                  "blobCount": 1,
+                  "blobName": "fsblob",
+                  "size": 4.4,
```

Comment:
Maybe clarify the meaning of these a bit?
"size" -> "totalFileSizeMB"
"rate" -> "filesPerSec"

Reply:
Jim's suggestions are fine with me.